include/exadg/postprocessor/kinetic_energy_calculation.cpp

```diff
@@ -47,6 +46,13 @@ KineticEnergyCalculator<dim, Number>::setup(dealii::MatrixFree<dim, Number> cons
   quad_index  = quad_index_in;
   data        = kinetic_energy_data_in;
 
+  time_control.reinit(kinetic_energy_data_in.calculate,
+                      0.0,                                // TODO: currently placeholder
+                      std::numeric_limits<double>::max(), // TODO: currently placeholder
+                      -1.0,                               // TODO:currently placeholder
```

Comment:
I think we should equip `kinetic_energy_data` with `start_time`, `time_interval`, etc.

The default setting for `kinetic_energy_data` should be to do it every time step, so that existing code is not changed.